pkg/util/clientauthorizer/admin.go

```diff
@@ -0,0 +1,31 @@
+package clientauthorizer
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"crypto/tls"
+
+	"github.com/sirupsen/logrus"
+)
+
+type admin struct {
+	log *logrus.Entry
+}
+
+func NewAdmin(log *logrus.Entry) ClientAuthorizer {
+	a := &admin{
+		log: log,
+	}
+
+	return a
+}
+
+func (a *admin) IsAuthorized(cs *tls.ConnectionState) bool {
+	a.log.Print("Admin auth is not implemented yet")
+	return false
+}
+
+func (a *admin) IsReady() bool {
+	return true
```

Comment:
`IsReady` always returns true?

Reply:
Yes. At least for now. This method indicates whether the authoriser is ready to make a decisions or not. In some cases (like in case of ARM authoriser in `arm.go`) authorisers need to fetch (for the first time or refresh) a certificate from a remote endpoint so it will be returning `false` while doing so.

This one is just a skeleton at themoment that declines all the admin requests and it's always ready.

Reply:
Gotcha, thanks